api/capi/src/nntrainer.cpp

```diff
@@ -216,7 +216,7 @@ int ml_train_model_compile(ml_train_model_h model, ...) {
   if (status != ML_ERROR_NONE)
     return status;
 
-  f = [&]() { return NN->checkValidation(); };
+  f = [&]() { return NN->isInitializable(); };
```

Comment:
I think this is for just checking the configuration before Initialize.

Reply:
Yes. It checks if it is able to initialize. So I clarified the name